client/app/vps/secondary-dns/vps-secondary-dns.html

```diff
@@ -1,27 +1,35 @@
 <section class="container-fluid cui-page__content cui-page__content-flex">
 
-    <button type="button"
-            class="oui-button oui-button_secondary"
-            data-translate="vps_configuration_secondarydns_add_title_button"
-            data-ng-click="$ctrl.add()"></button>
-
-    <oui-table rows="$ctrl.secondaryDns.list.results"
-               rows-loader="$ctrl.loaders.init">
-        <column title="'vps_tab_SECONDARY_DNS_table_domain' | translate" property="domain" sortable>
-            <span data-ng-bind="$row.domain"></span>
-        </column>
-        <column title="'vps_tab_SECONDARY_DNS_table_ip' | translate" property="ipMaster" sortable>
-            <span data-ng-bind="$row.ipMaster"></span>
-        </column>
-        <column title="'vps_tab_SECONDARY_DNS_table_date' | translate" property="creationDate" sortable>
-            <span data-ng-bind="$row.creationDate"></span>
-        </column>
-        <column title="'vps_tab_SECONDARY_DNS_table_dns' | translate" property="dns" sortable>
-            <span data-ng-bind="$row.dns"></span>
-        </column>
-        <column title="'common_actions' | translate"
-                class="oui-table__cell_action oui-table__cell_action-menu"
-                template="$ctrl.actionTemplate()">
-        </column>
-    </oui-table>
-</section>
\ No newline at end of file
+    <div data-ng-if="$ctrl.loaders.init"
+         class="text-center">
+        <oui-loader size="m" class="d-inline-block"></oui-loader>
+    </div>
+    <div data-ng-if="!$ctrl.loaders.init">
+        <button type="button"
+                class="oui-button oui-button_secondary"
+                data-translate="vps_configuration_secondarydns_add_title_button"
+                data-ng-click="$ctrl.add()"></button>
+        <oui-table data-ng-show="!$ctrl.loaders.init"
```

Comment:
So, finally the goal is to avoid using list view loader or not ?

Reply:
rows-loader= cannot take a boolean, it takes a function who will return return a promise in this format:
```
{
    data: page, // your data (an array)
    meta: {
        currentOffset, // page offset (from 0)
        pageCount, // number of pages
        totalCount, // number of items
        pageSize // page size (optional)
    }
}
```
As said by @neolitec, there's no easy way to display loading animation at first time list is visible when using rows=.  So yes, I choose to display tab loader instead of modifying oui-list-view.

Reply:
I agree.